docs/README.md

```diff
@@ -32,13 +32,17 @@ The list of scalers currently supported by Bellhop:
 
 ## Deploying Bellhop
 ### Steps to deploy Bellhop infrastructure:
-1. Deploy directly to Azure using the button below
+1. Deploy directly to Azure using the button below.
 
 [![Deploy To Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fbellhop%2Fmain%2Ftemplates%2Fazuredeploy.json/createUIDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fbellhop%2Fmain%2Ftemplates%2FcreateUiDefinition.json)
 
-2. Provide the required values in the deployment template, then click 'Review + Create'
+2. Provide the required values in the deployment template, then click 'Next' to proceed to the Advanced settings, or click 'Review + Create' to accept the default Bellhop values.
 
-![Bellhop Deployment](./images/deployment.png)
+![Bellhop Deployment](./images/deployment_basics.png)
+
+3. You are provided the option to override the default tag values that Bellhop uses to custom values of your choice. Additionally you can define a tag 'prefix' which will prepend all other tags. Click 'Review + Create' to proceed.
```

Comment:
I'd make the language easier:
-->
You can optionally override the default tag values that Bellhop uses to custom values of your choice. Additionally, you can define a tag 'prefix' which will prepend all other tags. Click 'Review + Create' to proceed.

Reply:
I'd agree this could be worded better

Reply:
I'm updating the wording as discussed above.